textdescriptives/tests/test_coherence.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+import pytest
+import spacy
+
+from textdescriptives.components import Coherence
+
+
+@pytest.fixture(scope="function")
+def nlp_small():
+    nlp = spacy.load("en_core_web_sm")
+    nlp.add_pipe("textdescriptives.coherence")
+    return nlp
+
+
+@pytest.fixture(scope="function")
+def nlp_large():
+    nlp = spacy.load("en_core_web_lg")
+    nlp.add_pipe("textdescriptives.coherence")
+    return nlp
+
+
+def test_coherence_integration(nlp_small):
+    assert "textdescriptives.coherence" == nlp_small.pipe_names[-1]
+
+
+def test_coherence_small_model(nlp_small):
+    doc = nlp_small(
+        "This is a short and simple sentence. Here is yet another one. We need quite a few before these coherences metrics make sense. Rambling, on and on."
+    )
+
+    assert doc._.coherence
+    assert doc._.first_order_coherence_values
+    assert doc._.second_order_coherence_values
+
+
+def test_coherence_large_model(nlp_large):
+    doc = nlp_large(
+        "This is a short and simple sentence. Here is yet another one. We need quite a few before these coherences metrics make sense. Rambling, on and on."
+    )
+
+    assert doc._.coherence
+    assert doc._.first_order_coherence_values
+    assert doc._.second_order_coherence_values
+
+
+def test_coherence_small_model_single_sentence(nlp_small):
```

Comment:
I would combine the tests.

```suggestion
def test_coherence(nlp):
```
Then if we wish to use mulitple model we can use the @pytest.fixture to avoid duplication